bin/configs/spring-cloud.yaml

```diff
@@ -4,6 +4,6 @@ library: spring-cloud
 inputSpec: modules/openapi-generator/src/test/resources/3_0/petstore.yaml
 templateDir: modules/openapi-generator/src/main/resources/JavaSpring
 additionalProperties:
-  documentationProvider: springfox
+  documentationProvider: springdoc
```

Comment:
seems to be similar to `bin/configs/spring-cloud-petstore-feign-spring-pageable.yaml`. IMHO: you can remove current one and leave only `bin/configs/spring-cloud-petstore-feign-spring-pageable.yaml`